tools/evals/baselines/factual_accuracy.yaml

```
@@ -433,3 +433,288 @@ samples:
+  - id: factual-accuracy-empty-12-none
+    human_score: none
+    inputs:
+      evaluated: "   "
+
+  - id: factual-accuracy-whitespace-none
+    human_score: none
+    inputs:
+      evaluated: |
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)